src/dotnet-scaffolding/dotnet-scaffold-aspnet/Templates/BlazorCrud/Index.cs

```diff
@@ -43,8 +43,8 @@ public virtual string TransformText()
 
             this.Write("@page \"/");
             this.Write(this.ToStringHelper.ToStringWithCulture(pluralModelLowerInv));
-            this.Write("\"\r\n\r\n@using Microsoft.AspNetCore.Components.QuickGrid\r\n@using Microsoft.EntityFra" +
-                    "meworkCore\r\n");
+            this.Write("\"\r\n@using Microsoft.AspNetCore.Components.QuickGrid\r\n@using Microsoft.EntityFrame" +
```

Comment:
Just curious about why a number of the lines are broken with concatenation (`+`) while other lines are broken across `this.Write()` calls?